Kernel/Arch/x86_64/init.cpp

```diff
@@ -177,12 +181,26 @@ extern "C" [[noreturn]] UNMAP_AFTER_INIT void init(BootInfo const& boot_info)
     // We need to copy the command line before kmalloc is initialized,
     // as it may overwrite parts of multiboot!
     CommandLine::early_initialize(kernel_cmdline);
-    memcpy(multiboot_copy_boot_modules_array, multiboot_modules, multiboot_modules_count * sizeof(multiboot_module_entry_t));
-    multiboot_copy_boot_modules_count = multiboot_modules_count;
 
     new (&bsp_processor()) Processor();
     bsp_processor().early_initialize(0);
 
+    if (multiboot_modules_count > 0) {
+        dbgln("Initramfs image {}, end {}", multiboot_module_ramdisk_physical_start, multiboot_module_ramdisk_physical_end);
```

Comment:
Hmm, we assume here that the multiboot module is always an initramfs image, is that a correct assumption?

Reply:
We could try to detect what is the multiboot module is about, but really we should only care about an initramfs and nothing else.

Reply:
I think we could do something about this, but for now let's keep it this way.